fastai/vision/learner.py

```diff
@@ -154,6 +154,7 @@ def _add_norm(dls, meta, pretrained):
     if not pretrained: return
     stats = meta.get('stats')
     if stats is None: return
+    if dls.after_batch.fs.filter(lambda tfm: isinstance(tfm, Normalize)): return
```

Comment:
```suggestion
    if dls.after_batch.fs.filter(risinstance(Normalize)):
```

https://fastcore.fast.ai/basics.html#risinstance